### DISCUSSION TITLE
Add `$emitDown` method for emitting events to nested child components

### DISCUSSION THREAD

**Discussion by ralphjsmit** [livewire/livewire#4204] [3. Feature ideas]
Hey there, I was wondering if would be possible to add some sort of `$this->emitDown($event)` method to the roadmap? Emit up is very useful, except that there are also use cases where you want to emit an event to nested components inside another component. Feels like a missing method, because we also have the `->up()` method.

I'm sorry, but I've almost no idea about the Livewire internals, so I don't know where to start with a PR.

Would this be possible to implement?

**Comment by joshhanley** [comment]
@ralphjsmit have a look at this open PR #3916 🙂
